src/document.h

```diff
@@ -76,9 +77,14 @@ struct Document {
     wxPrintData printData;
     wxPageSetupDialogData pageSetupData;
     uint printscale {0};
-    bool redrawpending {false};
     bool scrolltoselection {true};
     bool scaledviewingmode {false};
+    bool updatehover {false};

```

Comment:
would maybe be good to give all these variables the same prefix, so that it is clear these are all used for passing events to Draw